packages/AutoTDD-GUI.package/ATDDOverview.class/instance/testingEntityCollectionDictionary.st

```diff
@@ -0,0 +1,11 @@
+calculation
+testingEntityCollectionDictionary
+	
+	"EXACT COPY FROM ATDDBROWSER (where to put this (possibly along with watchers, gitHubActions, ...))"
```

Comment:
Is this comment still here intentionaly?

Reply:
Yeah, we still have to exclude this duplication, wrote it just so we don't foregt about it